skill-tree/src/main/java/com/RDS/skilltree/config/WebMvcConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.RDS.skilltree.Config;
```

Comment:
do we really need this file here

Reply:
I didn't fully understand. You mean this file should be somewhere else? Or this class isn't needed at all? 

Basically the interceptor I added to act as a middleware to check the request params before flow goes to the service keeping validation separate. We can discuss, I'm not fully clear here